bin/README.md

```diff
@@ -0,0 +1,146 @@
+# Machine setup
```

Comment:
This file doesn't belong in `bin/`. The only things that go in `bin/` are executable files. I suggest putting it in `requirements/` instead.

Then you can add another readme in `dot/` that goes over all the configuration files and what they're for, especially `.bashrc`.